CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # master
 
+# v2.1.0
+
+* Support rendering collections (e.g., `render(MyComponent.with_collection(@items))`).
+
+    *Tim Clem, Joel Hawksley*
```

Comment:
```suggestion
    *Tim Clem*
```

Reply:
I can't possible take enough credit to warrant putting my name on this 🙇

Reply:
LOL, fair enough. Thanks for getting me ramped up quickly!